### PR TITLE
Add sample and test for BoundingBoxToRect

### PR DESCRIPTION
**Pull request by yuyu2172**

### CODE REVIEW

**Comment by k-okada** [comment]
@yuyu2172 nice, please put images on this PR's comment that explains your work, like https://github.com/jsk-ros-pkg/jsk_recognition/pull/1563

**Comment by yuyu2172** [comment]
This example draws rectangles around six bins of APC shelf. 
The bounding boxes are estimated from a model of the shelf. 

bounding_box_to_rect_example

File: doc/jsk_perception/nodes/bounding_box_to_rect.md [hunk 1]
```diff
@@ -21,3 +21,8 @@ Convert `jsk_recognition_msgs/BoundingBoxArray` to `jsk_recognition_msgs/RectArr
   Projected 2-D bounding box.
 
 * `~internal` (`jsk_recognition_msgs/BoundingBoxArrayWithCameraInfo`)
+
+## Example
+![](images/bounding_box_to_rect_example.jpg)
+
+`roslaunch jsk_perception sample_bounding_box_to_rect.launch`
```
Comment (wkentaro) [inline doc/jsk_perception/nodes/bounding_box_to_rect.md]: Please add below:
```
rosrun jsk_perception install_test_data.py
```

**Comment by k-okada** [comment]
Is this ok to merge? or need to wait for https://github.com/jsk-ros-pkg/jsk_recognition/pull/1577#discussion_r58650102 ? @yuyu2172

**Comment by yuyu2172** [comment]
Yes, please merge.

**Comment by k-okada** [comment]
Could not merge because Travis failed, please look into them, or join slack to restart test if needed

--
◉ Kei Okada

2016/04/08 13:41、Yusuke Niitani <notifications@github.com> のメッセージ:

**Comment by yuyu2172** [comment]
I think it is ready to be merged.

**Pull request merged**